src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div>
+      <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post"
+        class="form "
+        data-qa="big"
+      >
+        <img
+          src="images/Search.svg"
+          alt="image"
+          class="img__search"
+          >
```

Comment:
```suggestion
        <img
          src="images/Search.svg"
          alt="image"
          class="img__search"
        >
```
closing bracket should be on the same level as opening one